<?xml version="1.0" ?>
<launch>
    <!-- Start print_node -->
    <node name="map_reader" pkg="pmadmu_planner" type="map_reader.py" output="screen" />
    <node name="transformation_server" pkg="pmadmu_planner" type="transformation_server.py" output="screen" />
    <node name="costmap_reader" pkg="pmadmu_planner" type="costmap_reader.py" output="screen" />
</launch>
